integration_tests/test_str_03.py

```diff
@@ -5,6 +5,14 @@ def test_new_line():
     print("\ndef")
     print("x\nyz")
 
+def test_escape_sequences():
+    print('I said \'Wow\' ')
+    print("I said \"hi\" ")
+    print("I said \\\"hi\" ")
+    print('I said \'\\\'hi\\''hey\' ')
+    print('a''b')
+    print("a" "b")
```

Comment:
Can you please add this test to `tests/parser/string2.py` as well? (so that we test the AST changes)